web/packages/extension/assets/_locales/sv/messages.json

```diff
@@ -94,5 +97,14 @@
     },
     "settings_saved": {
         "message": "Inställningar Sparade"
+    },
+    "permissions_grant": {
+        "message": "Ge Ruffle tillåtelse att få tillgång till min data på alla sidor"
+    },
+    "permissions_not_granted": {
+        "message": "Du behöver inte trycka på Ruffle ikonen på varje sida du vill använda Ruffle på"
```

Comment:
Huh, isn't this exactly the opposite of what it should be...?! As in, "don't have to", when it should be "have to"...?